_posts/2017-05-15-ge2017-brexit.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "BREXIT"
+authors:
+  - philip-john
+featured_image:
+  url: https://c1.staticflickr.com/5/4045/4606341298_ce0e5a94c0_z.jpg
+  alt: Europe
+  attribution: Europe by Niccolò Caranti (CC-BY-NC 2.0)
+  link: https://www.flickr.com/photos/ncaranti/4606341298/
+---
+
+{: .tldr}
+We don't think the UK should leave the EU - we're better off working together. If the UK does leave, we will aim to remain part of the EEA and EFTA. If we stay, we'll campaign hard for reform, greater democracy and more transparency.
+
+The 2017 election is being treated by some as a Brexit election. If that's so, we should take the opportunity to look closely at what happened with the referendum, and where we should go next.
+
+## Advisory
+
+First, we should remember that the referendum was advisory. Parliament still retains sovereignty and is not bound by the result of the referendum. In essence, Parliament asked the people for our views, but was not required to make any decision based on that referendum.
+
+## Lies
+
+There is no doubt that the campaigns peddled countless lies about what would happen in either scenario. On that basis alone, Parliament could reasonably choose not to proceed with Brexit, given many voters were duped. There have been numerous polls indicating differing outcomes were the referendum to be re-run and while these should all be taken with a pinch of salt there is a bad smell about the whole affair. This is not how democracy should be.
+
+## No clear outcome
+
+When asked to make our choice in the referendum, there was no plan presented for what a Leave vote would mean. Nor did the Remain campaign adequately address what it would do to address the EU's many failings. Had there been a guarantee of a referendum on the terms of Brexit in the event of a Leave vote or the reforms we would pursue under Remain, it would have given the electorate a much clearer indication of *what* they were voting for.
+
+Instead we now have politicians clambering to pursue their own agenda and claiming they have "the will of the people" or a "mandate" without having explicit public backing for their plans.
+
+## 37%
+
+Much is said about the 48%, but what about the 28% - the nearly 13m people who didn't vote? We cannot assume they were either Leave or Remain, and we should not ignore them.
+
+In fact, only 37% of the electorate (46,500,001) voted Leave. Such massive constitutional change should not be decided by a minority. We'd like to see referendums on serious constitutional change, such as Brexit, requiring 50% or more of the electorate (23,250,000 votes) to change the status quo.
```

Comment:
That 45 million figure is presumably the entire electorate? It reads like it's the 37% figure, to me...